.evergreen/run-unit-tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+############################################
+#            Main Program                  #
+############################################
+
+source java-config.sh
+
+echo "mongo-spring-session: running unit tests ..."
+
+./gradlew -version
+
+./gradlew -PjavaVersion=${JAVA_VERSION} --stacktrace --info --continue clean test
```

Comment:
do we really need a continue here ? 
I imagine if I am developing something and triggering pipeline , it would be great to have this pipeline fail as fast as possible because of my change 
But up to you , not sure if we use continue in java driver